src/ripple/app/paths/PathRequest.cpp

```diff
@@ -307,8 +307,7 @@ int PathRequest::parseJson (Json::Value const& jvParams)
         return PFR_PJ_INVALID;
     }
 
-    convert_all_ =
-        saDstAmount == STAmount(saDstAmount.issue(), 1u, 0, true);
+    convert_all_ = saDstAmount == STAmount(saDstAmount.issue(), 1u, 0, true);
```

Comment:
The comparison would be a lot cheaper if it was an `IOUAmount`. I don't think its a big enough deal to bother with now but when `IOUAmount` is more widespread we could revisit it.